lib/Data/Database.php

```diff
@@ -865,4 +973,25 @@ private static function _upgradeDatabase($oldversion)
                 );
         }
     }
+
+    /**
+     * read CLOB for OCI
+     * https://stackoverflow.com/questions/36200534/pdo-oci-into-a-clob-field
+     *
+     * @access private
+     * @static
+     * @param  resource $column
+     * @return string
+     */
+    private static function _clob($column)
```

Comment:
BTW uff crazy solution.

I also saw https://bugs.php.net/bug.php?id=57490 which BTW says "BLOB inserts working fine" and also uses OCI. Are you sure blobs do not work here for us as well?

Reply:
BLOB ≠ CLOB